worlds/Files.py

```diff
@@ -89,7 +96,15 @@ def read(self, file: Optional[Union[str, BinaryIO]] = None) -> None:
         with zipfile.ZipFile(zip_file, "r") as zf:
             if file:
                 self.path = zf.filename
-            self.read_contents(zf)
+            try:
+                self.read_contents(zf)
+            except Exception as e:
+                message = ""
+                if len(e.args):
```

Comment:
does `len` really make sense here over just `if e.args`?

Reply:
I dislike the python-ism of empty: false, non-empty:true
I've seen other people (in AP discord) say they dislike it also.
It doesn't match other languages, and can take people by surprise. So I prefer to avoid using it.